update-codacy-coverage.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+python-codacy-coverage -r coverage.xml
```

Comment:
What is the rationale behind having such a script for a single-command operation that's only ever going to be executed by the CI ? Why not just putting that command directly in the `.travis.yml` ?

Reply:
The file name would ideally be `travis-after-script.sh` to contain any future development of actions to be executed after Travis' validation.

Reply:
I'd rather simply use the feature of being able to define a list of commands in `travis.ini`'s after_script, we'll switch to a `travis-after-script.sh` if the need to use bash features arises.